crates/wasi/src/preview2/stdio.rs

```diff
@@ -22,7 +21,7 @@ pub use self::worker_thread_stdin::{stdin, Stdin};
 pub struct Stdout(AsyncWriteStream);
 
 pub fn stdout() -> Stdout {
-    Stdout(AsyncWriteStream::new(tokio::io::stdout()))
+    Stdout(AsyncWriteStream::new(2048, tokio::io::stdout()))
```

Comment:
Could 2048 be a documented constant here? That seems a little small to have as a budget, but we can always change it later.

Reply:
I upped it to 4096 because thats what we let blocking-write-and-flush accept